gpflow/optimizers/mcmc.py

```diff
@@ -41,41 +41,34 @@ class SamplingHelper:
 
         @tf.function
         def run_chain_fn():
-            return mcmc.sample_chain(num_samples, num_burnin_steps, current_state, kernel=adaptive_hmc)
+            return mcmc.sample_chain(
+                num_samples, num_burnin_steps, current_state, kernel=adaptive_hmc)
 
         hmc_samples = run_chain_fn()
         parameter_samples = hmc_helper.convert_samples_to_parameter_values(hmc_samples)
 
-    Args:
-        target_log_prob_fn: Python callable which represents log-density under the target distribution.
-        model_parameters: List of `tensorflow.Variable`s or `gpflow.Parameter`s used as a state of the Markov chain.
+    :param target_log_prob_fn: a callable which represents log-density under
+        the target distribution.
+    :param parameters: List of ``gpflow.Parameter`` used as a state of the Markov chain.
```

Comment:
Everywhere else we use a single back-tick for code, no?

```suggestion
    :param parameters: List of `gpflow.Parameter` used as a state of the Markov chain.
```